temporalio/client.py

```diff
@@ -1977,13 +1986,15 @@ async def _start_update(
         update: Union[str, Callable],
         arg: Any = temporalio.common._arg_unset,
         *,
+        wait_for_stage: WorkflowUpdateStage,
         args: Sequence[Any] = [],
         id: Optional[str] = None,
-        wait_for_stage: temporalio.api.enums.v1.UpdateWorkflowExecutionLifecycleStage.ValueType = temporalio.api.enums.v1.UpdateWorkflowExecutionLifecycleStage.UPDATE_WORKFLOW_EXECUTION_LIFECYCLE_STAGE_ADMITTED,
         result_type: Optional[Type] = None,
         rpc_metadata: Mapping[str, str] = {},
         rpc_timeout: Optional[timedelta] = None,
     ) -> WorkflowUpdateHandle[Any]:
+        if wait_for_stage == WorkflowUpdateStage.ADMITTED:
+            raise ValueError("ADMITTED wait stage not supported")
```

Comment:
Imo probably just let the server deal with this so we don't need a release to remove it later

Reply:
I think we want to revisit because I am hoping this required parameter becomes optional and durability + admitted becomes the default (like it is with signals).